scripts/pi-hole/js/messages.js

```diff
@@ -178,19 +166,21 @@ function delMsg(ids) {
         utils.showAlert(
           "success",
           "far fa-trash-alt",
-          "Successfully deleted " + ids.length + " message" + (ids.length > 1 ? "s" : ""),
-          ""
+          "Successfully deleted message",
+          "",
```

Comment:
As you said, a second option would be to change the title and also replace the empty `message` with something like this:

```suggestion
          "Success",
          "Message ID: " + id + " deleted.",
```